package tutorial;

message weather_message {
  required float datetime = 1;
  required float pressure = 2;
  required float altimeter = 3;
  required float outTemp = 4;
  required float outHumidity = 5;
  required float windSpeed = 6;
  required float windDir = 7;
  required float windGust = 8;
  required float windGustDir = 9;
  required float rainRate = 10;
  required float rain = 11;
  required float dewpoint = 12;
  required float modem_MB_usage = 13;
}

